setup.py

```diff
@@ -11,16 +11,19 @@
         ('share/ament_index/resource_index/packages',
             ['resource/' + package_name]),
         ('share/' + package_name + '/resource',
-            ['resource/gauges_2_widget.ui']),
+            ['resource/gauges_2.ui']),
         ('share/' + package_name, ['package.xml']),
         ('share/' + package_name, ['plugin.xml']),
     ],
     install_requires=['setuptools'],
+    zip_safe=True,
```

Comment:
What is this for?

Reply:
I'm not sure, it came as default with the command `ros2 pkg create --build-type ament_python <package_name>` so i left it.

Reply:
https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html
Just delete it

Reply:
Makes sense. Done.